tests/functional/TestKubeSmokeTest/testHeketiRpc.sh

```diff
@@ -90,7 +114,6 @@ setup_all_pods() {
 
 	echo -e "\nStart gluster mock container"
 	start_mock_gluster_container 1
-	start_mock_gluster_container 2
 }
 
 test_peer_probe() {
```

Comment:
So We do not label the node for DeamonSet to select? I know there is only one node and it will not make much difference as your finding pod doesn't check for node labels(yeah it is not necessary). Just wanted to clarify.